website/src/pages/account/index.tsx

```diff
@@ -1,9 +1,11 @@
-import { Button } from "@chakra-ui/react";
+import { Button, Divider, Flex, Grid, Icon, Text } from "@chakra-ui/react";
 import Head from "next/head";
 import Link from "next/link";
 import { useSession } from "next-auth/react";
 import React from "react";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
+import { MdOutlineEdit } from "react-icons/md";
```

Comment:
We are trying to use one type of icon. Can you switch to Feather icon here?

Reply:
I am not sure we talked about that.

Feather icon is not installed anyway, I will leave it to a different PR.

Reply:
I've been using Font Awesome (`react-icons/fa`) but they don't don't have an outline edit icon.  I think this is okay for now.